xbmc/cores/DataCacheCore.h

```diff
@@ -166,8 +163,48 @@ class CDataCacheCore
   mutable CCriticalSection m_contentSection;
   struct SContentInfo
   {
-    std::vector<EDL::Edit> m_editList;
-    std::vector<std::pair<std::string, int64_t>> m_chapters; // name and position for chapters
+  public:
+    /*!
+      * @brief Set the EDL edit list in cache.
+      * @param editList the list of edits to store in cache
+      */
+    void SetEditList(const std::vector<EDL::Edit>& editList) { m_editList = editList; }
+
+    /*!
+      * @brief Get the EDL edit list in cache.
+      * @return the list of edits in cache
+      */
+    std::vector<EDL::Edit> GetEditList() const { return m_editList; }
+
+    /*!
+      * @brief Save the chapter list in cache.
+      * @param chapters the list of chapters to store in cache
+      */
+    void SetChapters(const std::vector<std::pair<std::string, int64_t>>& chapters)
+    {
+      m_chapters = chapters;
+    }
+
+    /*!
+      * @brief Get the list of chapters in cache.
+      * @return the list of chapters in cache
+      */
+    std::vector<std::pair<std::string, int64_t>> GetChapters() const { return m_chapters; }
+
+    /*!
+      * @brief Reset the content cache to the original values (all empty)
+      */
+    void Reset()
+    {
+      m_editList.clear();
+      m_chapters.clear();
+    }
+
+  private:
+    /*!< list of EDL edits */
+    std::vector<EDL::Edit> m_editList{};
```

Comment:
No need for the initializer list. `std::vector<EDL::Edit> m_editList;`is sufficient as deafult ctor will be called automatically.